plenum/test/batching_3pc/test_basic_batching.py

```diff
@@ -56,14 +54,12 @@ def testTreeRootsCorrectAfterEachBatch(tconf, looper, txnPoolNodeSet,
     """
     # Send 1 batch
     reqs = sendRandomRequests(wallet1, client, tconf.Max3PCBatchSize)
-    waitForSufficientRepliesForRequests(looper, client, requests=reqs,
-                                        customTimeoutPerReq=tconf.Max3PCBatchWait)
+    waitForSufficientRepliesForRequests(looper, client, requests=reqs)
     checkNodesHaveSameRoots(txnPoolNodeSet)
 
     # Send 2 batches
     reqs = sendRandomRequests(wallet1, client, 2 * tconf.Max3PCBatchSize)
-    waitForSufficientRepliesForRequests(looper, client, requests=reqs,
-                                        customTimeoutPerReq=2*tconf.Max3PCBatchWait)
+    waitForSufficientRepliesForRequests(looper, client, requests=reqs)
     checkNodesHaveSameRoots(txnPoolNodeSet)
 
```

Comment:
No, and i dont see in the current setup if can test deterministically on the order of millisecond.